src/form-elements/FormElementSection.tsx

```diff
@@ -77,42 +77,76 @@ function FormElementSection<T extends FormTypes._NestedElementsElement>({
         'ob-section__valid': isValid,
       })}
     >
-      <div
-        className="ob-section__header cypress-section-header"
-        onClick={toggle}
-      >
-        <h3 className="ob-section__header-text title is-3">
-          {element.label}
-          {element.hint && (
-            <Tooltip
-              title={element.hint}
-              arrow
-              enterTouchDelay={0}
-              leaveTouchDelay={10000}
+      {element.hintPosition === 'TOOLTIP' || !element.hintPosition ? (
+        <div
```

Comment:
This seems like too much duplicated code, want to have another go at it, or pair it?